src/openjd/cli/_run/_run_command.py

```diff
@@ -117,60 +126,57 @@ def add_run_arguments(run_parser: ArgumentParser):
         "--env",
         dest="environments",
         action="append",
-        type=str,
         metavar="<path-to-JSON/YAML-file> [<path-to-JSON/YAML-file>] ...",
         help="Apply the given environments to the Session in the order given.",
     )
     run_parser.add_argument(
         "--preserve",
-        action="store_const",
-        const=True,
+        action="store_true",
         default=False,
         help="Do not automatically delete the Session's Working Directory when complete.",
     )
     run_parser.add_argument(
         "--verbose",
-        action="store_const",
-        const=True,
+        action="store_true",
         default=False,
         help="Enable verbose logging while running the Session.",
     )
+    run_parser.add_argument(
+        "--timestamp-format",
+        choices=["relative", "local", "utc"],
+        default="relative",
+        help="How to format the log output timestamps when running the job.",
+    )
+    run_parser.add_argument(
+        "--extensions",
+        help="A comma-separated list of Open Job Description extension names to enable. Defaults to all that are implemented.",
```

Comment:
Would it make sense to include the supported extensions in this help text?

Reply:
Yeah, that makes sense. I've added it in an update.